src/services/keycloak.js

```diff
@@ -3,24 +3,33 @@ import rp from 'request-promise-native'
 
 export default class KeycloakClient {
 
-    constructor( config ) {
+    constructor( config, logger ) {
```

Comment:
@Magnitus- I would just change the constructor to `( config, logger = null)` to make it more explicit that logger is not required.

Reply:
@nyanofthemoon Done. Thanks for the feedback!